### DISCUSSION TITLE
Dataset required for training

### DISCUSSION THREAD

**Discussion by Greyhat890** [coqui-ai/TTS#856] [General Q&A]
When I try to train my small dataset which contains 50 .wav files and 1 .csv file which is created using Mycroft recording studio.
When I start the training ..it pops out a warning/error which say " Required 100 files in dataset for training a model ".
How can I change this configuration ? which file is responsible  for this ? which parameter I want to change it ?
I am expecting to take 10-20 files of dataset for training a model. As I am new to this field. so anyone in this community help me to configure this parameter !... 
I know reducing the dataset file from 100 to 20 will not have a quality output.. I just wanted to try it out so please Anyone in this community help me to configure the paramter..

**Comment by erogol** [comment]
50 wav files are not enough to train a TTS model. You need at least around 3 hours of recordings. Then you can fine-tune one of our released models on your dataset. 

You can also try using SC-Glow model with your 50 clips but it needs some advanced TTS knowledge as we did not release an API yet for the use case.
